test/src/test/java/lib/form/AdvancedButtonTest.java

```diff
@@ -31,7 +31,7 @@ public class AdvancedButtonTest {
     public void testNestedOptionalBlock() throws Exception {
         HtmlPage page = j.createWebClient().goTo("self/testNestedOptionalBlock");
         HtmlForm form = page.getFormByName("config");
-        HtmlFormUtil.getButtonByCaption(form, "Advanced...").click();
+        HtmlFormUtil.getButtonByCaption(form, "Advanced").click();
```

Comment:
Is it not useful to use `…` as indicator that clicking does not result in an immediate change?

Reply:
There's a small icon in the button now rather than the three dots to let the user know that content will appear below.

![](https://user-images.githubusercontent.com/43062514/192627051-7adc9a7c-a892-4120-a297-76846ff23f13.png)